README.md

```diff
@@ -69,5 +79,5 @@ UserApp.add_all_users = function(data) {
 };
 ```
 
-
+-I know we just went over this but I'm having a massive brain fart, it has something to do with "users" and "data"
```

Comment:
Yup, definitely something to do with "users" and "data." The `UserApp.add_all_users(data);` line needs to be inside of the `.done()` callback in order to have access to `data`.